src/tabs/configuration.html

```diff
@@ -348,7 +348,7 @@
                                 </div>
                             </div>
                         </div>
-                        <div class="gui_box grey">
+                        <div class="gui_box grey accelDisabled">
```

Comment:
The class must define what it contains and not the opposite, so better `accelNeeded` or `accelEnabled`